content/components/toggle-switch.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Toggle switch
+description: A toggle switch is used to immediately toggle a setting on or off.
+---
+
+import {Box, Text} from '@primer/components'
+
+## Anatomy
+
+<Box display="flex" alignItems="flex-start" flexDirection={["column", "column", "column", "column", "row"]} sx={{gap: 3}}>
+    <Box as="ul" m="0" p="0">
+        <li><strong>Track</strong>: The clickable area where the toggle switch moves back and forth</li>
+        <li><strong>Knob</strong>: Changes color and position depending on whether the switch is on or off</li>
+        <li><strong>Status</strong>: A text label showing whether the switch is on or off</li>
+    </Box>
+    <img
+        width="456"
+        role="presentation"
+        src="https://user-images.githubusercontent.com/2313998/159587114-97d7de3c-9ff6-4f13-beca-6e4da2d548c6.png"
+    />
+</Box>
+
+## Contextual content
```

Comment:
This is all about labels, could the heading just be "Labels"?

I would also suggest that this section becomes part of the Options section, and that you move the Options section to here, so it's:
- Anatomy
- Options
  - Labels
  - Layout
  - Size
  - Loading
  - Progressive disclosure
- Mixed values
- Toggle switch vs checkbox
- Accessibility
  - Keyboard navigation
  - Touch target

Potentially **Accessibility** could be before **Mixed values** too.